### PR TITLE
Add statsd.

### PR DESCRIPTION
**Pull request by glogiotatidis**
Statsd setup and first metric.

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@pmclanahan r?

File: snippets/settings.py [hunk 1]
```diff
@@ -242,3 +244,23 @@ def mysql_version(self):
...
+        with open("/proc/net/route") as fh:
+            for line in fh:
+                fields = line.strip().split()
+                if fields[1] != '00000000' or not int(fields[3], 16) & 2:
+                    continue
+
+                return socket.inet_ntoa(struct.pack("<L", int(fields[2], 16)))
+    except IOError:
+        return 'localhost'
+
```
Comment (pmac) [inline snippets/settings.py]: NIT: One more blank line

**Comment by pmac** [comment]
r+

File: snippets/settings.py (hunk 1 above)
Comment (glogiotatidis) [inline snippets/settings.py]: flake8 doesn't care but i'll take your advice :)

**Pull request merged**